pkg/f1/testing/t.go

```diff
@@ -158,9 +167,18 @@ func (t *T) Fatalf(format string, args ...interface{}) {
 	t.FailNow()
 }
 
-// Fatal is equivalent to Log followed by FailNow.
-func (t *T) Fatal(err error) {
-	t.logger.Error("iteration failed", log.IterationAttr(t.Iteration), log.ErrorAttr(err))
+// Fatal logs an error then calls FailNow.
+func (t *T) Fatal(args ...interface{}) {
+	// Special case, single error argument
+	if len(args) == 1 {
+		err, ok := args[0].(error)
+		if ok {
+			t.logger.Error("iteration failed", log.IterationAttr(t.Iteration), log.ErrorAttr(err))
+			t.FailNow()
+			return
+		}
+	}
+	t.Log(args...)
```

Comment:
This would have different logging depending on the number of arguments.

I think if we just add a new log Attr helper that merges a list of errors, we can keep logic as is

Reply:
Okay I'm still not exactly clear on what you are looking for here. The original comment said `// Error is equivalent to Log followed by Fail.` (consistent with behaviour of testing.T) so I was aiming to get as close to that as possible.

Please could you clarify the behaviour you would like to see like to see with:
- a single argument of type `error`
- a single argument of type other than `error`
- more than one argument, all of type `error`
- more than one argument, not all of type `error`

I would also argue that rather than introducing a helper to log the arguments, we should amend `Log` so that it does what we want then use that in the methods. This simplifies code, makes behaviour consistent across these methods and  makes the original comments for `Error` and `Fatal` accurate.